internal/helper/helper.go

```diff
@@ -0,0 +1,49 @@
+package helper
+
+import (
+	"io"
+	"os"
+	"testing"
+)
+
+// CaptureStdout returns an io.Reader to read what was printed, and teardown
+func CaptureStdout(t *testing.T) (io.Reader, func()) {
```

Comment:
To my eyes there is a 3 line difference between the Stdout and Stderr functions, if we can it'd be nice to merge those and we can leave the existing CaptureStd{out,err} functions here as the convenient API that calls to a common function.